lib/doorman.ex

```diff
@@ -1,2 +1,48 @@
 defmodule Doorman do
+  @moduledoc """
+  Provides helpers that take advantage of the options configured in your config
+  files.
+  """
+
+  @doc """
+  Authenticates a user by their email and password. Returns true user is found
```

Comment:
I think "Returns true user is found" should be "Returns true if the user is found"